conda_build/index.py

```diff
@@ -781,12 +781,20 @@ def __init__(self, channel_root, channel_name, subdirs=None, threads=MAX_THREADS
         self.deep_integrity_check = deep_integrity_check
 
     def index(self, patch_generator, hotfix_source_repo=None, verbose=False, progress=False,
-              current_index_versions=None, index_file=None):
+              current_index_versions=None, index_file=None, append=None):
         if verbose:
             level = logging.DEBUG
         else:
             level = logging.ERROR
 
+        # dict(subdir: repodata)
```

Comment:
Mind elaborating a bit in the comment what this is used for?

Reply:
Was just keeping notes, I'm happy to remove that.